runtime/src/module.c

```diff
@@ -182,29 +183,34 @@ module_new(char *name, char *path, uint32_t stack_size, uint32_t max_memory, uin
 	admissions_info_initialize(&module->admissions_info, admissions_percentile, expected_execution,
 	                           module->relative_deadline);
 
+	/* WebAssembly Indirect Table */
+	/* TODO: Should this be part of the module or per-sandbox? */
+	/* TODO: How should this table be sized? */
```

Comment:
These TODOs seem to relate to discussions around the "store" and module instantiation, so likely worth deferring.